### ISSUE TITLE
About fine-grained ground truth class

### ISSUE THREAD

**Issue by xiaopang114** [doantienthongbku/Implementation-patchnet#7]
Hello,
I hope you're doing well. I've been using your fantastic project, and I have a question about fine-grained ground truth class. 

I noticed in your paper that you mentioned redefining the dataset into finer-grained labels based on devices and materials, such as dividing the labels of Oulu dataset's protocol 1 into 5 categories. However, I actually did not see any code regarding multi-class labels and  how to convert multi-class labels into binary labels. Many demonstrations still show binary classification problems, for example, 
1. In the readme file, you wrote 'with [set_name.csv] have format (label only has 2 classes: 0-Spoofing, 1-Liveness):', 
2. In the Implementation-patchnet/metrics/losses.py file: self.amsm_loss = AdMSoftmaxLoss(1024, 2), self.m = [m_s, m_l], m = torch.tensor([self.m[ele] for ele in labels]).to(x.device)

I look forward to your reply.
